examples/python/copy_stencil.py

```diff
@@ -0,0 +1,94 @@
+"""Copy stencil HIR generator
```

Comment:
I would generally advise to follow https://www.python.org/dev/peps/pep-0008/ as a python style guide unless you have another preference

Reply:
mostly https://www.python.org/dev/peps/pep-0008/#naming-conventions:

Classes are CamelCase, functions are lowercase with underscores , variables are also lowercase with underscores

Reply:
done